tests/async/models.py

```diff
@@ -1,5 +1,16 @@
 from django.db import models
+from django.utils import timezone
+
+
+class RelatedModel(models.Model):
+    simple = models.ForeignKey("SimpleModel", models.CASCADE, null=True)
 
 
 class SimpleModel(models.Model):
     field = models.IntegerField()
+    created = models.DateTimeField(default=timezone.now)
+
+
+class Book(models.Model):
+    title = models.CharField(max_length=200)
+    pages = models.IntegerField(default=0)
```

Comment:
with the tests merged, there’s no need to create a new model, we can reuse `SimpleModel`. Each model increases the runtime a little bit of the test suite as its table needs to be created. We try to avoid this where possible.

```suggestion
```